### DISCUSSION TITLE
Using multivelo on WNN_umap

### DISCUSSION THREAD

**Discussion by FayKong123** [welch-lab/MultiVelo#24] [Q&A]
Hi, 
Thanks for your package.  I am trying to apply it on my 10x RNA+ATAC. but we already finished the processing of data using Seurat/Signac, and we already used WNN to intergrate RNA and ATAC. Now we want to try multivelo on our WNN umap, is that possible? or do we have to reprocessing our data from the beginning(CellRanger)?
Best wishes.

**Comment by danielee0707** [comment]
Hi. You'll need the original 10X BAM file or Fastq files in order to quantify the unspliced and spliced counts. If you already have them, after finishing all the preprocessing steps in python, you can replace the adata_rna.obsm['X_umap'] field with the WNN umap and run multivelo. Or alternatively, you can replace it after running multivelo and use it for plotting directly, and the result will be slightly different.
